apps/storybook-react-native/stories/WalletHome.stories.tsx

```diff
@@ -0,0 +1,409 @@
+import React from 'react';
```

Comment:
Example of design system react native components being used in a wallet home page example